### PR TITLE
Prime: document Tallon Canyon and Gully

### PR DESCRIPTION
**Pull request by JeffGainsNGames**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added: In Super Missile Room: Intermediate Single-wall Wall Jump to reach Tunnel to Super Missile Room Access. 
 
+### Metroid Prime
+
+#### Logic Database
+
+##### Tallon Overworld
+
+- Added: In Tallon Canyon: Intermediate Standable, Beginner BSJ, Intermediate Dash, Intermediate Slope Jump to climb to top of half-pipe.
```
Comment (Miepee) [inline CHANGELOG.md:37]:
```suggestion
- Added: In Tallon Canyon: Climb to the top of the half-pipe via one of the following: Intermediate Standable, Beginner BSJ, Intermediate Dash, Intermediate Slope Jump.
```

Sounds nicer IMO.

File: CHANGELOG.md [hunk 2]
```diff
@@ -28,6 +28,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Added: In Tallon Canyon: Beginner Knowledge to break blocks with Power Bombs.
+- Added: In Tallon Canyon: Advanced Wall Boost to climb room from Gully.
+- Added: In Gully: Beginner Standables to climb room.
```
Comment (Miepee) [inline CHANGELOG.md:40]:
```suggestion
- Added: In Gully: Beginner Standables to climb the room.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -28,6 +28,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+### Metroid Prime
+
+#### Logic Database
+
+##### Tallon Overworld
+
+- Added: In Tallon Canyon: Intermediate Standable, Beginner BSJ, Intermediate Dash, Intermediate Slope Jump to climb to top of half-pipe.
+- Added: In Tallon Canyon: Beginner Knowledge to break blocks with Power Bombs.
+- Added: In Tallon Canyon: Advanced Wall Boost to climb room from Gully.
```
Comment (Miepee) [inline CHANGELOG.md:39]:
```suggestion
- Added: In Tallon Canyon: Advanced Wall Boost to climb the room from Gully.
```

File: randovania/games/prime1/logic_database/Tallon Overworld.txt [hunk 4]
```diff
@@ -260,7 +277,29 @@ Extra - aabb: [-579.22754, 296.85858, -35.60154, -454.3111, 409.0403, 42.921246]
...
+                  Space Jump Boots
+                  Any of the following:
+                      # https://youtu.be/lN6aTh05TgA
+                      Standable Terrain (Intermediate)
+                      # https://youtu.be/K2hpJEYeIeg
+                      Combat/Scan Dash (Intermediate)
+                      # https://youtu.be/X-pKDXxZh6s
+                      Morph Ball Bomb and Morph Ball and Bomb Space Jump (Beginner)
+                      # https://youtu.be/lN6aTh05TgA
+                      R-Jump (Beginner) and Slope Jump (Intermediate)
```
Comment (Miepee) [inline randovania/games/prime1/logic_database/Tallon Overworld.txt:294]: the video here lists the same video as for standables. doesnt seem intentional.

File: randovania/games/prime1/logic_database/Tallon Overworld.txt [hunk 5]
```diff
@@ -260,7 +277,29 @@ Extra - aabb: [-579.22754, 296.85858, -35.60154, -454.3111, 409.0403, 42.921246]
...
+              All of the following:
+                  Space Jump Boots
+                  Any of the following:
+                      # https://youtu.be/lN6aTh05TgA
+                      Standable Terrain (Intermediate)
+                      # https://youtu.be/K2hpJEYeIeg
+                      Combat/Scan Dash (Intermediate)
+                      # https://youtu.be/X-pKDXxZh6s
+                      Morph Ball Bomb and Morph Ball and Bomb Space Jump (Beginner)
+                      # https://youtu.be/lN6aTh05TgA
```
Comment (Samuel6710) [inline randovania/games/prime1/logic_database/Tallon Overworld.txt]: Looks like you linked the incorrect video here.

File: randovania/games/prime1/logic_database/Tallon Overworld.txt [hunk 6]
```diff
@@ -260,7 +277,29 @@ Extra - aabb: [-579.22754, 296.85858, -35.60154, -454.3111, 409.0403, 42.921246]
...
+                      R-Jump (Beginner) and Slope Jump (Intermediate)
+          All of the following:
+              # Method to get past blocks
+              Morph Ball
+              Any of the following:
+                  Morph Ball Bomb
+                  Power Bomb and Knowledge (Beginner)
+                  # https://youtu.be/IHV-S_1WM9s
+                  Boost Ball and Space Jump Boots and Single Room Out of Bounds (Intermediate) and R-Jump (Intermediate) and Standable Terrain (Intermediate)
```
Comment (Samuel6710) [inline randovania/games/prime1/logic_database/Tallon Overworld.txt:302]: You'll need `Clip Through Objects` for this trick. I'd probably say Advanced is good.

**Review by Samuel6710 — CHANGES_REQUESTED** [review]

File: randovania/games/prime1/logic_database/Tallon Overworld.txt (hunk 6 above)
Comment (JeffGainsNGames) [reply randovania/games/prime1/logic_database/Tallon Overworld.txt:302]: advanced? it's not too bad, and I never boosted back OoB in attempts. fine with either int or advanced but wanted to discuss

Comment (JeffGainsNGames) [reply randovania/games/prime1/logic_database/Tallon Overworld.txt:302]: I guess transport access OoB is adv, so fine with that

**Review by Samuel6710 — APPROVED** [review]

**Comment by Miepee** [comment]
merging next release as we are in feature freeze and this isnt urgent.

**Pull request merged**